### PR TITLE
[TASK] Update System modules section

### PR DESCRIPTION
**Pull request by linawolf**
Releases: main, 13.4, 12.4

### CODE REVIEW

File: Documentation/Concepts/Backend/SystemModules/Index.rst [hunk 1]
```diff
@@ -1,77 +1,113 @@
...
 .. include:: /Includes.rst.txt
-.. index:: Modules; System
 .. _system-modules:
 
 ==============
 System modules
 ==============
 
-.. important::
+The backend modules in group system are only available to backend users with
```
Comment (Alagts) [inline Documentation/Concepts/Backend/SystemModules/Index.rst:9]:
```suggestion
System modules are backend modules in the group "System" 
and they are only available to backend users with
```

File: Documentation/Concepts/Backend/SystemModules/Index.rst [hunk 2]
```diff
@@ -1,77 +1,113 @@
...
+admin permissions.
```
Comment (Alagts) [inline Documentation/Concepts/Backend/SystemModules/Index.rst:11]:
```suggestion

In TYPO3 you can decide which package you want to include in your 
TYPO3 instance.
```

File: Documentation/Concepts/Backend/SystemModules/Index.rst [hunk 3]
```diff
@@ -1,77 +1,113 @@
...
 
-   This chapters cover modules that will only be available for backend users
-   with admin access privileges.
+Some modules are only available when you have an optional system extension
```
Comment (Alagts) [inline Documentation/Concepts/Backend/SystemModules/Index.rst:12]: What is meant by "optional system extension" here?

File: Documentation/Concepts/Backend/SystemModules/Index.rst [hunk 4]
```diff
@@ -1,77 +1,113 @@
...
+.. _system-modules-permissions:
 
-.. index:: Modules; Log
-.. _system-modules-log:
+Permissions
+===========
 
-Log
-===
+The module :guilabel:`System > Permissions` can be used to view or edit
```
Comment (Alagts) [inline Documentation/Concepts/Backend/SystemModules/Index.rst:23]:
```suggestion
In TYPO3, you can grant permissions to backend users. 
At first, a newly created backend user without any administrative 
privileges has no access to neither the page module nor the 
pages in the backend.    
The module :guilabel:`System > Permissions` can be used to view or edit
```

File: Documentation/Concepts/Backend/SystemModules/Index.rst [hunk 5]
```diff
@@ -1,77 +1,113 @@
...
+backend user permissions for pages in the backend.
```
Comment (Alagts) [inline Documentation/Concepts/Backend/SystemModules/Index.rst:24]:
```suggestion
these backend user permissions for pages in the backend.
```

File: Documentation/Concepts/Backend/SystemModules/Index.rst [hunk 6]
```diff
@@ -1,77 +1,113 @@
...
 
-The *Database (DB) check* module offers four functions related
-to the database and its content.
+Webhooks (optional)
+===================
+
+This module is only available if the system extension
+:composer:`typo3/cms-webhooks` is installed. This extension handles **outgoing**
+webhooks to TYPO3. It also provides a corresponding backend module to manage
+webhook records in the TYPO3.
```
Comment (Alagts) [inline Documentation/Concepts/Backend/SystemModules/Index.rst:61]:
```suggestion
webhook records in TYPO3.
```

File: Documentation/Concepts/Backend/SystemModules/Index.rst [hunk 7]
```diff
@@ -1,77 +1,113 @@
...
-  Checks if certain relations are empty or broken, typically
-  used to check if files are being referenced.
+    Checks if certain relations are empty or broken, typically
+    used to check if files are being referenced.
 
 Search
-  A tool to search through the whole database. It offers an
-  advanced mode which is similar to a visual query builder.
+    A tool to search through the whole database. It offers an
+    advanced mode which is similar to a visual query builder.
```
Comment (Alagts) [inline Documentation/Concepts/Backend/SystemModules/Index.rst:98]: What is a visual query builder? I would add an explanation here.

Comment (linawolf) [reply Documentation/Concepts/Backend/SystemModules/Index.rst:103]: We can enhance descriptions in a follow-up

File: Documentation/Concepts/Backend/SystemModules/Index.rst (hunk 3 above)
Comment (linawolf) [reply Documentation/Concepts/Backend/SystemModules/Index.rst:13]: One that you can install but don't have to install. Maybe put it in the glossary: 

Optional system extension: A system extension in TYPO3 that is not installed by default but can be installed as needed to provide additional functionality.

File: Documentation/Concepts/Backend/SystemModules/Index.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/Concepts/Backend/SystemModules/Index.rst:12]: This is out of context here as we are describing the modules functionality. I would describe such somewhere in the extensions chapteres

**Pull request merged**